fast-project-initializer/Assets/_Scripts/Editor/UI/Views/DirectoryInitializerView.cs

```diff
@@ -0,0 +1,16 @@
+using UnityEngine;
+
+namespace PushForce.FastProjectInitializer.UI
+{
+	public class DirectoryInitializerView : IView
+	{
+		public void drawGUI()
+		{
+			GUI.Label(new Rect(25, 75, 200, 50), "Test Label");
+			if (GUI.Button(new Rect(25, 25, 200, 50), "Restart"))
```

Comment:
keys

Reply:
Those keys will disappear in the future so i will leave them as they are for now.